src/api/apiBase.ts

```diff
@@ -34,7 +34,6 @@ const apiBase = (withVersion = true) => {
 export default apiBase;
 
 export function termsBase() {
-  // TODO: This needs to handle local vs ferdium vs franz servers
   return (window as any).ferdium.stores.settings.all.app.server !== LIVE_FRANZ_API
```

Comment:
Do you think this if condition itself can be removed?

Reply:
I don't think so:

-  it handles a difference on the URLs from https://api.getferdi.com or https://api.ferdium.org, where we store our terms file (in /terms)
- The DEV_API_FRANZ_WEBSITE is https://meetfranz.com

Reply:
ok  - i havent delved too deep into this right now (have to start work meetings). I will resolve this comment. I also haven't run this locally, if you are confident, please squash and merge